data/hai/hai outfits.txt

```diff
@@ -693,4 +693,5 @@ outfit "Failed Control System"
 	thumbnail "outfit/control transceiver"
 	"mass" 1
 	"unplunderable" 1
-	description "This was meant to control some sort of experimental system, but it appears to have failed."
+	"integrated control systems" 1
+	description "This was meant to control some sort of experimental system, but it appears to have failed. Unfortunately it doesn't appear to be safely removable."
```

Comment:
It's still here. :(

Reply:
Well you never did answer the questions posed to state outright whether you just wanted it to check for the ship or if you wanted it removed entirely. 
We did give a number of reasons for not wanting it removed entirely which you never actually addressed/responded to.

Reply:
At least now its not longer bugged

Reply:
The stated purpose of the outfit is in its use as a thematic element and plot hook. Now how does it do that? Well it's unique, it's out of place, it draws the eye, its existence begs a question. But are these things not also true of the ship itself? Barring those aspects of the outfit that overlap with the ship, what do we actually get out of this outfit that we don't already have? An outfit description where the ship lacks a visible one? Can that same information not be portrayed via a mission conversation or log entry? Or can we not wait on ships which aren't being sold for their descriptions to be visible, as we had waited on better autoconditions to improve this mission already? My point is I find the outfit redundant and unnecessary.

Reply:
I personally like the outfit a lot for asking questions. The Geocoris itself is simple: bigger, weaponised and with an inbuilt syscore. Nothing mysterious about that if you ask me.

Reply:
Well I do think the unique Geocoris sprite could use some work. Could be make more interesting/mysterious.

Reply:
But why would the Geocoris itself look more interesting/mysterious?
The only thing really mysterious about is *interior* to the ship.
Hence, the outfit to represent the curiosity on the interior; one that you can't natively remove.